components/sections/Footer/index.module.css

```diff
@@ -0,0 +1,38 @@
+.footer {
+  @apply flex
+    flex-col
+    items-center
+    gap-6
+    px-8
+    md:flex-row
+    md:justify-between;
```

Comment:
Can you check the design again? Border top, vertical padding for mobile, and maximum width for desktop should be added on the footer

Reply:
added vertical padding in mobile & desktop mode